app/src/main/res/values/dimens.xml

```diff
@@ -3,17 +3,28 @@
     <dimen name="dp_50">50dp</dimen>
     <dimen name="dp_8">8dp</dimen>
     <dimen name="dp_5">5dp</dimen>
+    <dimen name="dp_2">2dp</dimen>
     <dimen name="dp_10">10dp</dimen>
     <dimen name="dp_16">16dp</dimen>
     <dimen name="dp_24">24dp</dimen>
     <dimen name="dp_22">22dp</dimen>
+    <dimen name="dp_36">36dp</dimen>
     <dimen name="dp_40">40dp</dimen>
+    <dimen name="dp_45">45dp</dimen>
     <dimen name="dp_52">52dp</dimen>
     <dimen name="dp_85">85dp</dimen>
+    <dimen name="dp_70">70dp</dimen>
     <dimen name="dp_122">122dp</dimen>
     <dimen name="dp_133">133dp</dimen>
-    <dimen name="sp_20">20sp</dimen>
+    <dimen name="sp_20">18sp</dimen>
+    <dimen name="sp_16">16sp</dimen>
     <dimen name="sp_22">22sp</dimen>
     <dimen name="sp_50">50sp</dimen>
     <dimen name="sp_90">90sp</dimen>
+    <dimen name="dp_30">30dp</dimen>
+    <dimen name="sp_65">65sp</dimen>
+    <dimen name="dp_35">35dp</dimen>
+    <dimen name="image_view">110dp</dimen>
+    <dimen name="fab_margin">16dp</dimen>

```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.fab_margin` appears to be unused
 Unused resources make applications larger and slow down builds.